test/integration/smoke/test_vm_life_cycle.py

```diff
@@ -1443,6 +1443,12 @@ def setUp(self):
         if len(self.hosts) < 2:
             self.skipTest("Requires at least two hosts for performing migration related tests")
 
+
+        for host in self.hosts:
```

Comment:
@DaanHoogland it seems not be a CentOS issue, but issue with test scripts or libvrt/kvm plugin.
let's skip the live migration with volume on kvm at first, so trillian test for other prs is good.

we can fix the issue afterwards. it needs investigation.

could you change the lines to skip the test by checking the host hypervisor instead of host os ?

Reply:
I could, but since the tests are for KVM only anyway, that would effectively mean disabling the tests completely.